src/jukebox/components/battery_monitor/batt_mon_i2c_ina219/__init__.py

```diff
@@ -0,0 +1,83 @@
+# RPi-Jukebox-RFID Version 3
+# Copyright (c) See file LICENSE in project root folder
+
+import logging
+import jukebox.plugs as plugs
+import jukebox.cfghandler
+from ina219 import INA219
+from ina219 import DeviceRangeError
+from components.battery_monitor import BatteryMonitorBase
+
+logger = logging.getLogger('jb.battmon.ina219')
+
+batt_mon = None
+
+
+class battmon_ina219(BatteryMonitorBase.BattmonBase):
+    '''Battery Monitor based on a INA219
+
+    CAUTION - WARNING
```

Comment:
I suggest removing the docs from the py file as you already wrote the documentation in a md file. This would avoid duplicate information, which needs to maintained.